lib/ash/actions/managed_relationships.ex

```diff
@@ -2015,5 +2263,52 @@ defmodule Ash.Actions.ManagedRelationships do
     |> Ash.Changeset.set_context(relationship.context)
     |> Ash.Changeset.set_tenant(tenant)
     |> Ash.destroy(return_notifications?: true)
+    |> case do
+      {:ok, notifications} ->
+        debug_log(relationship.destination, changeset, :destroy, :ok, opts[:debug?])
+        {:ok, notifications}
+
+      {:error, error} ->
+        debug_log(
+          relationship.source,
+          changeset,
+          :destroy,
+          {:error, error},
+          opts[:debug?]
+        )
+
+        {:error, error}
+    end
+  end
+
+  defp debug_log(relationship, changeset, action, response, debug?) do
```

Comment:
This `relationship` argument is actually `destination` now, but I think we should just pass down the relationship *name* being edited, since there could be multiple relationships to the same destination being edited at one time. I think once we rename the argument it should be good to go 😄

Reply:
Like this:
```
13:09:38.944 [debug] Managed Relationship Debug, Ash.Test.ManageRelationshipTest.RelatedResource:parent_resource with action create_with_parent: Create success.
```
?

Reply:
Above is currently committed. 
But I also tried this (not committed):
```
13:34:56.493 [debug] [manage_relationship: :parent_resource] [action: :create_with_existing_parent] [resource: Ash.Test.ManageRelationshipTest.RelatedResource] [response: :ok]
```
in the format of `[debug]` wdyt? error message is in it's own line and it's `[response: :error]`.
This might be later colorized 🤠  but in general I think it would be cool to be able debug any change for any action for any resource and a consistent format around it would be also cool; like a more technical and direct log right... In the end it would be great to turn on a lot of `debug?` flags and still be able to figure out what's what easily from the logs.

Reply:
I think for now I prefer the current format. If we were to do that, it would be `[resource: ..., response: ...]` in a single keyword list probably. I do see your point about having some kind of standardized format etc, but I think that can be its own conversation.

Reply:
So then is this it? 😆